02_Data_Management/Layers_and_Table_Views.Rmd

```diff
@@ -130,78 +175,3 @@ data("badeplaetze_zh")
 
 ```{r, echo=F}
 # Municipality (Gemeinde) of Richterswil 
-richterswil <- filter(gemeinde_zh, gemeindena == "Richterswil")
-names(richterswil)[5] = "Municipality"
-
-ggplot() + geom_sf(data = gemeinde_zh) + geom_sf(data = richterswil,aes(fill = Municipality))
-+ geom_sf(data = badeplaetze_zh,color = "blue") + ggtitle("Swimming spots in the Canton of Zurich") + theme_minimal() + coord_sf(datum = 2056)
-```
-
-
-```{r}
-swimmSpots_richt <- badeplaetze_zh[richterswil, ,op = st_within]
-```
-
-```{r,echo=F}
-ggplot() + geom_sf(data = gemeinde_zh) + geom_sf(data = badeplaetze_zh,color = "blue") + geom_sf(data = richterswil, fill = "white") + geom_sf(data = swimmSpots_richt, color = "red") + ggtitle("Swimming spots in the Canton of Zurich") + theme_minimal() + coord_sf(datum = 2056)
-```
-
-
-
-### Completely_within
```

Comment:
if you removed these subchapters, where are they now?

Reply:
@ratnanil  based on my approach, these are now under the umbrella of Layers and Table Views -> Select by Location -> Completely within (as #### subchapter)
<